README.md

```diff
@@ -44,6 +44,14 @@ WORKING_DIR        | no        | c:\libro-sync-agent\working
 API_ENDPOINT       | no        | https://api.libroreserve.com/inbound/maitre_d/status |
 STRIP_INVOICE_DATA | no        | 0                                                    |
 
+### Cohabitation
+
+If the Restock agent is running on the server (the Restock logo will appear on the system tray), do not modify the previous POS configuration. On the Restock agent, you will need to activate the recopy service and set the directory location to `c:\libro-sync-agent\working` as seen in the screenshot below.
```

Comment:
- [ ] Remplacer "do not modify the previous POS configuration" par "do not modify the existing POS configuration"
- [ ] Metre "do not modify the existing POS configuration" en gras